app/views/ptables/index.html.erb

```diff
@@ -9,20 +11,26 @@
       <th><%= sort :name, :as => s_("Ptable|Name") %></th>
       <th><%= sort :family, :as => s_("Ptable|Os family") %></th>
```

Comment:
Could you change the translation in locale/en/foreman.po to "OS families" please?  (English doesn't go through Transifex.)